test_collections/matter/sdk_tests/scripts/fetch_sdk_tests_and_runner.sh

```diff
@@ -37,7 +37,7 @@ SDK_DATA_MODEL_PATH="src/app/zap-templates/zcl/data-model/chip"
 
 ENV_FILE="$ROOT_DIR/.env"
 
-TEST_COLLECTIONS_PATH="$ROOT_DIR/test_collections"
+TEST_COLLECTIONS_PATH="$ROOT_DIR/matter"
```

Comment:
I believe that the ROOT_DIR variable should have one more "/.." and this variable be "MATTER_TEST_COLLECTIONS_PATH =$ROOT_DIR/test_collections/matter"

Reply:
Done 

TEST_COLLECTIONS_PATH removed
ROOT_DIR is now **.../test_collection/matter**